### PR TITLE
Update dependencies and fix build

### PR DESCRIPTION
**Pull request by schillic**
Commits
1. update dependencies (needed to check out this package locally)
2. remove broken model from build (#82)
3. copy over `.nnet` files for `Literate` processing
4. remove broken model from build (algorithm crashes)

### CODE REVIEW

**Review by mforets — APPROVED** [review]

File: docs/generate.jl [hunk 1]
```diff
@@ -7,14 +7,21 @@ MODELS = [
...
     joinpath(@__DIR__, "..", "models", "Double Pendulum"),
     joinpath(@__DIR__, "..", "models", "VertCAS")
 ]
 GENERATEDDIR = joinpath(@__DIR__, "src", "models")
 mkpath(GENERATEDDIR)
 for model in MODELS
+    # copy over .nnet files
+    for file in readdir(model)
+        if any(endswith.(file, [".nnet"]))
+            cp(joinpath(model, file), joinpath(GENERATEDDIR, file); force=true)
```
Comment (mforets) [inline docs/generate.jl:21]: it is also possible to store auxiliary files in an `examples` folder, see for example here  https://github.com/JuliaReach/ReachabilityAnalysis.jl/blob/master/examples/ISS/ISS.jl#L34

either way is fine i guess

**Review by mforets — APPROVED** [review]

File: Project.toml [hunk 2]
```diff
@@ -12,12 +12,12 @@ ReachabilityAnalysis = "1e97bd63-91d1-579d-8e8d-501d2b57c93f"
...
 [compat]
-LazySets = "1.37"
-MAT = "0.8, 0.9"
+LazySets = "1"
+MAT = "0.8, 0.9, 0.10"
 MathematicalSystems = "0.11"
 NeuralVerification = "0"
-ReachabilityAnalysis = "0.7, 0.8, 0.9"
-Reexport = "0.2, 1.0"
+ReachabilityAnalysis = "0.7, 0.8, 0.9, 0.10"
```
Comment (mforets) [inline Project.toml:19]: let me add the Julia bots so this is done automatically

File: docs/generate.jl (hunk 1 above)
Comment (schillic) [reply docs/generate.jl:21]: I am also fine with both options.

File: Project.toml (hunk 2 above)
Comment (schillic) [reply Project.toml:19]: It is there (see the other PRs, which I only saw later) :wink:

Comment (mforets) [reply Project.toml:19]: indeed! :smile:

**Pull request merged**